### PR TITLE
[usage] Add GetStripeCustomer to BillingService

### PR DESCRIPTION
**Pull request by easyCZ**
## Description

This will allow us to move the lookup of customers to a single place and improve our ability to respond to it consistently.

## Related Issue(s)

* Part of https://github.com/gitpod-io/gitpod/issues/13177

## How to test

builds

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

File: components/usage-api/usage/v1/billing.proto [hunk 1]
```diff
@@ -35,3 +39,19 @@ message CancelSubscriptionRequest {
...
+}
+
+message GetStripeCustomerResponse {
+  StripeCustomer customer = 1;
+  string attribution_id = 2;
+}
+
+message StripeCustomer {
+  string id = 1;
+}
```
Comment (andrew-farries) [inline components/usage-api/usage/v1/billing.proto:57]: Is it worth inlining this into the `GetStripeCustomerResponse`?

**Review by andrew-farries — APPROVED** [review]
✅ just one question

File: components/usage-api/usage/v1/billing.proto (hunk 1 above)
Comment (easyCZ) [reply components/usage-api/usage/v1/billing.proto:57]: I'm keen for this to be extended in the future more fields, so would like to keep it as a dedicated object.

**Pull request merged**